maidsafe/safe_browser
# refactor(notifications): deprecates Notifier component in favor of antd

- Changed: HTTP requests are no longer automatically forwarded to
    default browser
  - Added: Initial HTTP allowed-list containing 'https://invite.maidsafe.net'
  - Removed: Notifier component
  - Changed: clearNotification operations may be passed notification ID,
    otherwise defaults to shift notification array
  - Added: Static Notification type declaration
  - Changed: Updates relevant tests which cover notifications

Resolves #619 



QA:
- HTTP requests either via address bar or via Fetch API should NOT forward to system's default browser. Instead, notification should appear, providing user with option to copy to clipboard.
- One exception is that `https://invite.maidsafe.net` has been added to an allowed list of acceptable HTTP requests, so that URL should be forwarded to default browser.
- Make a non-allowed HTTP request such as: `https://www.rust-lang.org` and observe notification
- Make an `index.html` which fetches non-allowed resources and open in browser, such as:
```html
<html>
    <body>
        <img src="https://i2.wp.com/www.foodrepublic.com/wp-content/uploads/2013/09/glamorous-assorted-apples-pictures-hd-wallpaper-hd-pictures.jpg"> 
        <img src="https://img1.cookinglight.timeinc.net/sites/default/files/styles/4_3_horizontal_-_1200x900/public/image/2017/11/main/satsuma-oranges-getty-1711w.jpg"> 
    </body>
</html>
```
Notifications will NOT appear for these two HTTP resources. Two errors will be logged to the console.
